commands/git-update

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+##
+# Usage
+#
+# Updates current branch based on its name:
+# $ git update
+#
+# Updates with rebase rather than merge
+# $ git update rebase
+##
+
+git fetch > /dev/null 2>&1
+DEVELOP_EXISTS=$(git ls-remote | grep '/heads/develop' | wc -l)
+
+# exit on error
+set -e
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+TYPE=$1
+
+
+if [[ $BRANCH = 'master' ]]; then
+  echo "you can't update master silly"
+  exit 255
+fi
+
+git fetch  # get new stuff
+
+if [[ ${BRANCH:0:8} = "feature/" || $DEVELOP_EXISTS -eq 1 ]]; then
+ if [[ $TYPE = 'rebase' ]]; then
+    git rebase origin/develop
+  else
+    git merge --no-ff origin/develop
```

Comment:
In these cases I think we *don't* want the merge commit, and fast-forwarding is good, so leave off the `--no-ff`